test/main.d

```diff
@@ -0,0 +1,26 @@
+import accessors;
+
+void main()
+{
+}
+
+// Issue #11: https://github.com/funkwerk/accessors/issues/11
+pure nothrow @safe @nogc unittest
```

Comment:
_Attributes should be listed in alphabetical ordering, e.g. `const @nogc nothrow pure @safe` (the ordering should ignore the leading `@`)._

Reply:
fixed.